pyunfold/teststat.py

```diff
@@ -206,16 +206,15 @@ def TSCalc(self, N1, N2):
         cs1 = np.cumsum(N1)/n1
         cs2 = np.cumsum(N2)/n2
 
-        NE = n1 * n2 / (n1 + n2)
-
         len1 = len(N1)
-        en = np.sqrt(len1/2)
+        self.en = np.sqrt(len1/2)
         d = np.max(np.abs(cs1-cs2))
+        self.d = d
         self.SetStat(d)
 
     def Prob(self):
         try:
-            prob = kstwobign.sf((en+.12+.11/en)*d)
+            prob = kstwobign.sf((self.en + .12 + .11 / self.en) * self.d)
```

Comment:
@zhampel just checking that these should be `self.en` and `self.d` instead of `en` and `d`. It kept throwing an error because the local variables `en` and `d` wheren't defined.